product_docs/docs/mongo_data_adapter/5/04_installing_the_mongo_data_adapter/x86_amd64/01_mongo_rhel8_x86.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Installing MongoDB Foreign Data Wrapper on RHEL 8/OL 8 x86"
+title: "Installing Replication Server on RHEL 8/OL 8 x86"
```

Comment:
Missed this on the first pass - this is definitely still the MongoFDW install instructions, not EPRS - best revert :)